ee/tpmrunner/tpmrunner_test.go

```diff
@@ -0,0 +1,80 @@
+package tpmrunner
```

Comment:
Same as for the secureenclaverunner -- would you paste in a `Test_InterruptMultiple` test?

Reply:
done, though I had to make the `interrupted` bools atmoic to deal with data race

Reply:
The data races with the rungroup actors are always such a pain in tests 😅